molecule/scenario_resources/Dockerfile.j2

```diff
@@ -61,10 +61,8 @@ systemctl enable firewalld
 # fedora or centos-8
 
 # Pulp requires CentOS 8.2 or later. 8.2 is currently CentOS Stream.
-RUN grep -v "CentOS" /etc/redhat-release || dnf install -y centos-release-stream epel-release &&\
```

Comment:
Do we use/test CentOS Stream anywhere?

Reply:
Thanks for clarification!

Reply:
Also, to answer your question directly, we will not use or test CentOS Stream anywhere once this PR is merged. We do want to at least test our installer against it because it is the future of CentOS/RHEL 8.x.